pkg/cmd/roachtest/tests/large_schema_benchmark.go

```diff
@@ -151,11 +151,16 @@ func registerLargeSchemaBenchmark(r registry.Registry, numTables int, isMultiReg
 						c.Start(ctx, t.L(), startOpts, settings, c.CRDBNodes())
 						conn := c.Conn(ctx, t.L(), 1)
 						defer conn.Close()
+						// Disable autocommit before DDL since we need to batch statements
+						// in a single transaction for them to complete in a reasonable amount
+						// of time. In multi-region this latency can be substantial.
+						_, err := conn.Exec("SET CLUSTER SETTING sql.defaults.autocommit_before_ddl.enabled = 'false'")
```

Comment:
this won't change the setting in the current session. i think that's fine since the table setup happens in a different connection, but just wanted to confirm.